kvmagent/kvmagent/test/localstorage_testsuite/test_data_volume_with_iothreadpin.py

```
@@ -212,34 +220,34 @@ def test_virtio_and_virtio_scsi_data_volume_with_iothreadpin(self):
+        vm_utils.del_iothread_pin(self.vm_uuid, virtio_scsi_vol_iothread_id)
+        vm_utils.del_iothread_pin(self.vm_uuid, virtio_vol_iothread_id)
 
-        rsp = vm_utils.check_volume(vm_uuid, [virtio_vol, virtio_scsi_vol])
+        rsp = vm_utils.check_volume(self.vm_uuid, [virtio_vol, virtio_scsi_vol])
         self.assertFalse(rsp.success)
 
-        vm_utils.destroy_vm(vm_uuid)
-        pid = linux.find_vm_pid_by_uuid(vm_uuid)
+        vm_utils.destroy_vm(self.vm_uuid)
+        pid = linux.find_vm_pid_by_uuid(self.vm_uuid)
         self.assertTrue(not pid, 'vm[%s] vm still running' % vm_uuid)
```

Ruff (0.15.7)

[error] `vm_uuid` may be undefined, or defined from star imports

(F405)